src/main/java/com/thoughtworks/gauge/command/SetupCommand.java

```diff
@@ -0,0 +1,110 @@
+                + "        vowels = new HashSet<>();\n"
+                + "        for (char ch : vowelString.toCharArray()) {\n"
+                + "            vowels.add(ch);\n"
+                + "        }\n"
+                + "    }\n"
+                + "\n"
+                + "    @Step(\"The word <word> has <expectedCount> vowels.\")\n"
+                + "    public void verifyVowelsCountInWord(String word, int expectedCount) {\n"
+                + "        int actualCount = countVowels(word);\n"
+                + "        assertThat(expectedCount).isEqualTo(actualCount);\n"
+                + "    }\n"
+                + "\n"
+                + "    @Step(\"Almost all words have vowels <wordsTable>\")\n"
+                + "    public void verifyVowelsCountInMultipleWords(Table wordsTable) {\n"
+                + "        for (TableRow row : wordsTable.getTableRows()) {\n"
+                + "            String word = row.getCell(\"Word\");\n"
+                + "            int expectedCount = Integer.parseInt(row.getCell(\"Vowel Count\"));\n"
+                + "            int actualCount = countVowels(word);\n"
+                + "\n"
+                + "            assertThat(expectedCount).isEqualTo(actualCount);\n"
+                + "        }\n"
+                + "    }\n"
+                + "\n"
+                + "    private int countVowels(String word) {\n"
+                + "        int count = 0;\n"
+                + "        for (char ch : word.toCharArray()) {\n"
+                + "            if (vowels.contains(ch)) {\n"
+                + "                count++;\n"
+                + "            }\n"
+                + "        }\n"
+                + "        return count;\n"
+                + "    }\n"
+                + "}";
+
+        String projectRoot = System.getenv("GAUGE_PROJECT_ROOT");
+
+        Path implFilePath = Paths.get(projectRoot, "src", "test", "java", "StepImplementation.java");
+        Logger.info(String.format("create %s", implFilePath.toString()));
+        this.writeContent(implFilePath, implementation);
+
+        Path propFilePath = Paths.get(projectRoot, "env", "default", "java.properties");
+        Logger.info(String.format("create %s", propFilePath.toString()));
+        this.writeContent(propFilePath, properties);
+
+        Path gitIgnoreFilePath = Paths.get(projectRoot, ".gitignore");
+        Logger.info(String.format("create %s", gitIgnoreFilePath.toString()));
+        this.writeContent(gitIgnoreFilePath, gitIgnore);
+
+        Path libsDirPath = Paths.get(projectRoot, "libs", ".gitkeep");
+        Logger.info(String.format("create %s", libsDirPath.toString()));
+        this.writeContent(libsDirPath, "");
+    }
+
+    private void writeContent(Path implFilePath, String content) throws IOException {
+        FileOutputStream fileOutputStream = FileUtils.openOutputStream(implFilePath.toFile(), true);
+        fileOutputStream.write(content.getBytes());
+    }
```

Comment:
Logging statement can be moved to the `writeContent` method.

>      private void writeContent(Path implFilePath, String content) throws IOException 

`templateFilePath` will be more appropriate name for `implFilePath`